index.js

```diff
@@ -9,6 +9,7 @@ const loadProbot = appFn => {
   probot = probot || createProbot({
     id: process.env.APP_ID,
     secret: process.env.WEBHOOK_SECRET,
+    webhookProxy: process.env.WEBHOOK_PROXY_URL,
```

Comment:
Support the smee proxy. (Should we actually port all the config from probot-run ?) https://github.com/probot/probot/blob/master/bin/probot-run.js#L24-L31